demos/src/inline.mustache

```diff
@@ -0,0 +1,34 @@
+<div class="demo-message">
+	<div class="o-message o-message--inline o-message--alert-success" data-o-component="o-message">
+		<div class="o-message__container">
+			<div class="o-message__content">
+				<p class="o-message__content--detail"><span class="o-message__content--highlight">Well done!</span> You have successfully completed this task and this is great news.</p>
+			</div>
+			<p class="o-message__content--action">Now you should do this, and this, and this, and this.</p>
```

Comment:
This text is still part of the content. It will (probably) be giving the user extra information, or next steps, but as text is all it is, the user can't interact with it. 

The actions in this case are the elements that a user _can_ interact with, such as a button or a link or a checkbox.

This probably means that we'll have to rejig the markup, to something that looks like:

```
<div class="o-message__content">
   <span class="o-message__content--highlight>Words</span
   <p class="o-message__content--info> More words</p>
   <p class="o-message__content--detailed-info> Even more words</p>
</div>
```
(naming things is hard 🤔 )

This will likely affect the layouts of the already existing elements, but hey! It's fine for things to change 💪